packages/terra-core-docs/src/terra-dev-site/doc/button/AnchorButton.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import { Badge } from 'terra-button/package.json?dev-site-package';
+
+import ButtonAnchor from './example/ButtonAnchor?dev-site-example';
+import ButtonPropsTable from 'terra-button/src/variants/_AnchorButton?dev-site-props-table';
+
+<Badge />
+
+# Anchor Button
+
+The subcomponent of [Terra-Button](https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/button/about) that can display a button rendered using anchor element.
+
+The `action` variant is intended for specific solutions in which a non-inline floating button sits above the page content in a fixed position.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+- `npm install terra-button`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
+import AnchorButton from 'terra-button/lib/variants/_AnchorButton';
```

Comment:
lets update this import statement to 
`import { AnchorButton } from 'terra-button';`

Reply:
updated in https://github.com/cerner/terra-core/pull/3655/commits/81401caf927c1b11a30aa72ed0740f8e7e2c6df6